test/gke/release-cluster.sh

```diff
@@ -7,7 +7,7 @@ project="cilium-ci"
 # this is only needs to be set as some of gcloud commands requires it,
 # but as this script uses resource URIs clusters in all locations are
 # going to be discovered and used
-region="us-west2"
+region="us-west1"
```

Comment:
This actually shouldn't matter, as location is part of URIs that get returned by the first call. The only reason this seems to be needed looks like a bug in `gcloud` flag parser... just saying :)

Reply:
I think it mattered for retrieving the node pools on line 37, but maybe it was caused by off-by-one error